rfd/0135-kube-apps-discovery.md

```diff
@@ -0,0 +1,199 @@
+---
+    authors: Anton Miniailo (anton@goteleport.com)
+    state: draft
+---
+
+# RFD 135 - Kubernetes App discovery
+
+## Required Approvers
+
+- Engineering: `@r0mant` && `@tigrato`
+- Product: `@klizhentas` || `@xinding33`
+- Security: `@reedloden` || `@jentfoo`
+
+## What
+
+Proposes discovery of apps that are hosted inside Kubernetes clusters and exposed as services.
+
+Related issue: [#25538](https://github.com/gravitational/teleport/issues/25538)
+
+## Why
+
+Some users dynamically deploy apps into their Kubernetes clusters and expose those as Kubernetes services. When they actively
+create/update/remove apps, manually trying to mirror these changes to Teleport may be time-consuming and prone to
+errors, especially for organizations that have a large number of services running in their Kubernetes clusters.
+With the changes described in this RFD, Teleport will be able to discover Kubernetes service and automatically register them as
```

Comment:
I'd remove `With the changes described in this RFD`, the sentence will still be valid.